models.py

```diff
@@ -179,8 +179,8 @@ class Player(BasePlayer):
 
     _gender =  models.StringField(
```

Comment:
Why changing the choices syntax? Have you tested if the experiment keeps running properly after this change?

Reply:
The form fields still work with the _(). The _() are the translations tags for python code